sagemaker-python-sdk/tensorflow_serving_container/tensorflow_serving_container.ipynb

```diff
@@ -0,0 +1,343 @@
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Optional: add a second model\n",
+    "\n",
+    "The TensorFlow Serving container can host multiple models, if they are packaged in the same model archive file. Let's prepare a second version of the MobileNet model so we can demonstrate this. The `mobilenet_v2_035_224` model is a shallower version of MobileNetV2 that trades accuracy for smaller model size and faster computation, but has the same inputs and outputs."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "second_model_name = 'mobilenet_v2_035_224'\n",
+    "second_model_path = sample_utils.tfhub_to_savedmodel(second_model_name, export_path)\n",
+    "\n",
+    "print('SavedModel exported to {}'.format(second_model_path))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Next we need to create a model archive file containing the exported model."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Create a model archive file\n",
+    "\n",
+    "SageMaker models need to be packaged in `.tar.gz` files. When your endpoint is provisioned, the files in the archive will be extracted and put in `/opt/ml/model/` on the endpoint. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!tar -C \"$PWD\" -czf mobilenet.tar.gz mobilenet/"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Upload the model archive file to S3\n",
+    "\n",
+    "We now have a suitable model archive ready in our notebook. We need to upload it to S3 before we can create a SageMaker Model that. We'll use the SageMaker Python SDK to handle the upload."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from sagemaker.session import Session\n",
+    "\n",
+    "model_data = Session().upload_data(path='mobilenet.tar.gz', key_prefix='model')\n",
+    "print('model uploaded to: {}'.format(model_data))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Create a SageMaker Model and Endpoint\n",
+    "\n",
+    "Now that the model archive is in S3, we can create a Model and deploy it to an \n",
+    "Endpoint with a few lines of python code:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from sagemaker.tensorflow.serving import Model\n",
+    "\n",
+    "# Use an env argument to set the name of the default model.\n",
+    "# This is optional, but recommended when you deploy multiple models\n",
+    "# so that requests that don't include a model name are sent to a \n",
+    "# predictable model.\n",
+    "env = {'SAGEMAKER_TFS_DEFAULT_MODEL_NAME': 'mobilenet_v2_140_224'}\n",
```

Comment:
how do users know to set this? are there other env variables they could set?

Reply:
they know because we tell them here, and in the sdk docs:

https://github.com/aws/sagemaker-python-sdk/blob/master/src/sagemaker/tensorflow/deploying_tensorflow_serving.rst#deploying-more-than-one-model-to-your-endpoint

there is one other env var to do with http error log verbosity, but I want to hold off on documenting until we've had some user feedback.

Reply:
okay. I think I'd add a link to the docs in this section that says this is where you can learn about environment variables to set (or something else vague enough that won't be wrong if you do decide to document more of these)